app/src/main/java/com/sh1r0/noveldroid/SettingsFragment.java

```diff
@@ -81,10 +86,42 @@ public void onCreate(Bundle paramBundle) {
 		}
 		aboutMessage = getString(R.string.version_tag) + versionName + "\n" + getString(R.string.author_tag) + getString(R.string.author_name);
 
-		namingRuleList = this.getResources().getStringArray(R.array.naming_rule);
-
 		encoding = findPreference(KEY_ENCODING);
 		namingRule = findPreference(KEY_NAMING_RULE);
+		namingRule.setOnPreferenceClickListener(new OnPreferenceClickListener() {
+			@Override
+			public boolean onPreferenceClick(Preference preference) {
+				LayoutInflater factory = LayoutInflater.from(getActivity());
+				final View dialog_view = factory.inflate(R.layout.naming_rule_dialog,null);
+				final EditText editText;
+				editText =  (EditText) dialog_view.findViewById(R.id.editText);
+				editText.setText(prefs.getString(KEY_NAMING_RULE, "!n"));
```

Comment:
Having a constant variable for this default value should provide more flexibility.